reference/5.1/Microsoft.PowerShell.Utility/Write-Error.md

```diff
@@ -361,6 +361,10 @@ a terminating error condition. To signal a terminating error, use the
 
 [about_Automatic_Variables](../microsoft.powershell.core/about/about_automatic_variables.md)
 
+[about_Output_Streams](/powershell/module/Microsoft.PowerShell.Core/About/about_Output_Streams.md)
```

Comment:
**Site Relative** links shouldn't include the extension.

Reply:
fixed